### PR TITLE
Fix strikethrough and typo in relationshipType translation table (Annex A)

### PR DESCRIPTION
**Pull request by bact**
- Remove strikethrough formatting
- Fix typo: hasPrequisite -> hasPrerequisite

### Questions

1. Does this table complete?
    - A sentence before the table said:
      > Relationship migration is being worked out in the relationships spreadsheet. Once completed, the following table will reflect the translation for relationship types from SPDX 2.3 to SPDX 3.0:
1. What does "lifecycle scope" in the last column mean?
    - Other values in this column are types in LifecycleScopeType (for example, "build", "build, runtime", "test") or a string stating "various LifecycleScopeType".
    - Should this "lifecycle scope" string be replaced by "various LifecycleScopeType"?
1. The 3rd column ("Swap to and from?") probably not complete?
    - For example, 2.3 `DATA_FILE_OF` and 3.0 `hasDataFile` they are swapped in direction. But there's no "Y" marked in their column.
    - A (is a) `DATA_FILE_OF` B
    - B `hasDataFile` A

### CODE REVIEW

File: docs/annexes/diffs-from-previous-editions.md [hunk 1]
```diff
@@ -222,10 +222,10 @@ Relationship migration is being worked out in the relationships spreadsheet.  On
 | COPY_OF | copiedTo | | |
 | DATA_FILE_OF | hasDataFile | | |
 | DEPENDENCY_MANIFEST_OF | hasDependencyManifest | | |
-| ~~DEPENDENCY_OF~~ | [removed] | | |
+| *DEPENDENCY_OF* | [removed] | | |
```
Comment (zvr) [inline docs/annexes/diffs-from-previous-editions.md:225]:
```suggestion
| DEPENDENCY_OF | [removed] | | |
```

File: docs/annexes/diffs-from-previous-editions.md [hunk 2]
```diff
@@ -222,10 +222,10 @@ Relationship migration is being worked out in the relationships spreadsheet.  On
...
 | DEPENDS_ON | dependsOn | | various LifecycleScopeType |
 | DESCENDANT_OF | decendentOf | | |
-| ~~DESCRIBED_BY~~ | [removed] | | |
+| *DESCRIBED_BY* | [removed] | | |
```
Comment (zvr) [inline docs/annexes/diffs-from-previous-editions.md:228]:
```suggestion
| DESCRIBED_BY | [removed] | | |
```

File: docs/annexes/diffs-from-previous-editions.md [hunk 3]
```diff
@@ -239,14 +239,14 @@ Relationship migration is being worked out in the relationships spreadsheet.  On
...
-| HAS_PREREQUISITE | hasPrequisite | | lifecycle scope |
+| HAS_PREREQUISITE | hasPrerequisite | | lifecycle scope |
 | METAFILE_OF | hasMetadata | | |
 | OPTIONAL_COMPONENT_OF | hasOptionalComponent | | |
 | OPTIONAL_DEPENDENCY_OF | hasOptionalDependency | | lifecycle scope |
 | OTHER | other | | |
 | PACKAGE_OF | packagedBy | | |
-| ~~PATCH_FOR~~ | [removed] | | |
-| ~~PREREQUISITE_FOR~~ | [removed] | | |
+| *PATCH_FOR* | [removed] | | |
```
Comment (zvr) [inline docs/annexes/diffs-from-previous-editions.md:248]:
```suggestion
| PATCH_FOR | [removed] | | |
```

File: docs/annexes/diffs-from-previous-editions.md [hunk 4]
```diff
@@ -239,14 +239,14 @@ Relationship migration is being worked out in the relationships spreadsheet.  On
...
+| *PREREQUISITE_FOR* | [removed] | | |
```
Comment (zvr) [inline docs/annexes/diffs-from-previous-editions.md:249]:
```suggestion
| PREREQUISITE_FOR | [removed] | | |
```

**Review by zvr — CHANGES_REQUESTED** [review]
I don't see a point in changing the typography for relationship names, since the second column clearly says they were removed.

**Comment by bact** [comment]
Close this one as it will be better covered by #1019

**Pull request closed without merging**